README.markdown

```diff
@@ -122,6 +124,8 @@ If you have many functions that go well together, you can put them as a `abcyzea
 
 If you would like to override the functionality of a plugin distributed with Oh My Zsh, create a plugin of the same name in the `custom/plugins/` directory and it will be loaded instead of the one in `plugins/`.
 
+Feel free to add your theme to the [wiki](https://github.com/robbyrussell/oh-my-zsh/wiki/External-themes), if you want to share it with others.
```

Comment:
I believe this reference is not really necessary, which would cut it down to two.